compiler/rustc_lint/src/lints.rs

```diff
@@ -674,6 +674,9 @@ pub struct DropCopyDiag<'a> {
     pub arg_ty: Ty<'a>,
     #[label]
     pub label: Span,
+    #[suggestion(style = "verbose", code = "{replace}", applicability = "maybe-incorrect")]
+    pub sugg: Option<Span>,
+    pub replace: String,
```

Comment:
I would prefer if we not print the note and the suggestion at the same time. Something like this should do it.

```suggestion
    #[subdiagnostic]
    pub sugg: DropCopySuggestion,
}

#[derive(Subdiagnostic)]
pub enum DropCopySuggestion {
    #[note(lint_dropping_copy_types_note)]
    Note {
        #[label]
        label: Span,
    }
    #[suggestion(lint_dropping_copy_types_suggestion, style = "verbose", code = "{replace}", applicability = "maybe-incorrect")]
    Suggestion {
        #[primary_span]
        sugg: Span,
        replace: String,
    }
}
```

Reply:
Done. Thank you.